src/main/java/org/mockito/internal/creation/MockSettingsImpl.java

```diff
@@ -95,8 +98,9 @@ public MockSettings defaultAnswer(Answer defaultAnswer) {
         return this;
     }
 
-    public MockSettings useConstructor() {
+    public MockSettings useConstructor(Object... constructorArgs) {
```

Comment:
@mureinik 
We should ensure that the vararg-array is not null. You can use the class Checks here.